pkgs/shelf_test_handler/mono_pkg.yaml

```diff
@@ -17,3 +17,5 @@ stages:
     os:
     - linux
     - windows
+  - test: --test-randomize-ordering-seed=random -p chrome -c dart2wasm
```

Comment:
Should we explicitly run this on linux and windows?

Reply:
This just defaults to linux IIUC, which is fine.